src/libraries/System.ComponentModel.Composition/tests/System/ComponentModel/Composition/CompositionExceptionTests.cs

```diff
@@ -416,8 +416,8 @@ private void AssertMessage(CompositionException exception, int rootCauseCount, C
                 }
                 else
                 {
-                    string option1 = string.Format(CultureInfo.CurrentCulture, SR.CompositionException_SingleErrorWithMultiplePaths, rootCauseCount);
-                    string option2 = string.Format(CultureInfo.CurrentCulture, SR.CompositionException_MultipleErrorsWithMultiplePaths, rootCauseCount);
+                    string option1 = SR.Format(SR.CompositionException_SingleErrorWithMultiplePaths, rootCauseCount);
```

Comment:
These are tests.  Do we use SR elsewhere in tests?

Reply:
In `main` branch, these test files are using SR.Format:
```sh
$ git grep -l 'SR.Format(' ':/*[tT]est*'
src/libraries/Common/tests/Tests/System/Net/aspnetcore/Http2/HPackDecoderTest.cs
src/libraries/Microsoft.Extensions.Configuration.Binder/tests/ConfigurationBinderTests.cs
src/libraries/Microsoft.Extensions.Configuration.Binder/tests/ConfigurationCollectionBindingTests.cs
src/libraries/Microsoft.Extensions.Configuration.CommandLine/tests/CommandLineTest.cs
src/libraries/Microsoft.Extensions.Configuration.Ini/tests/IniConfigurationTest.cs
src/libraries/Microsoft.Extensions.Configuration.UserSecrets/tests/ConfigurationExtensionTest.cs
src/libraries/Microsoft.Extensions.Configuration.Xml/tests/XmlConfigurationTest.cs
src/libraries/System.Net.Http/tests/UnitTests/HPack/HPackRoundtripTests.cs
```